privateigentum.md

```diff
@@ -1,44 +1,41 @@
-    nicht. Niemand will mir meine Zahnbürste oder meine Bettdecke
-    wegnehmen.
+        * Das ist schön wenn mir die Sachen gehören die ich benutzen will. Dafür braucht es aber die alleinige Verfügungsgewalt über Dinge nicht. Niemand will mir meine Zahnbürste oder meine Bettdecke wegnehmen.
+	        * Es gibt auch den Fall dass mir Dinge gestohlen werden die ich brauche (z.B. Auto etc.). Diese Dinge werden aber gerade gestohlen WEIL sie Privateigentum sind, niemand klaut ein Auto weil er etwas zum fahren braucht sondern weil er es für Geld verkaufen will.
 	    * Privateigentum bedeutet aber auch, dass mir die Sachen gehören müssen, die ich benutzen will.
 	    * Will ich also Sachen in Besitz nehmen, die mir nicht gehören, so muss ich den Menschen denen die Sachen gehören, entschädigen.
 	    * Gilt Privateigentum, bin ich prinzipiell von den Dingen die ich brauche und die mir nicht gehören getrennt.
     
-    1. Nicht immer gehören die Dinge die ich benutze auch wirklich mir. Beispiel: Mietwohnungen.
+    1. ### Nicht immer gehören die Dinge die ich benutze auch wirklich mir. Beispiel: Mietwohnungen.
 	    * Anders als bei der Zahnbürste fällt die Benutzung und das Eigentum über die Sache bei der Mietwohnung auseinander.
 	    * Ich muss jeden Monat Miete zahlen, um die Eigentümerin zu kompensieren.
 	    * Die Vermieterin hat also einen Vorteil aus ihrem Eigentum: Weil es ihr gehört, es aber andere brauchen, erhält sie ein Einkommen.
 	    * Da in diesem Verhältnis zwei entgegengesetzte Interessen vorhanden sind, braucht es Gewalt um es zu erhalten.
 	    * Der Staat hat Institutionen wie Polizei, Gerichte, Gefängnisse, Katasteramt, usw. welche die Eigentumsordnung durchzusetzen.
     
-    2. Es gibt auch eine dritte Art von Eigentum: Produktives Eigentum, welches zur Herstellung von Dingen oder Dienstleistungen
-    notwendig ist.
+    2. ### Es gibt auch eine dritte Art von Eigentum: Produktives Eigentum, welches zur Herstellung von Dingen oder Dienstleistungen notwendig ist.
 	    * Das sind z.B. Fabriken, Containerschiffe, Telefonleitungen, Minen, Büros, usw.
 	    * Dieses Eigentum begegnet den meisten Menschen in der Form von Arbeitsplätzen.
 	    * Diese Menschen brauchen Geld und können sich das nur verdienen, indem ihnen Zugriff auf das produktive Eigentum erlaubt wird (Fließband, Büro, Supermarktkasse).
-	    * Das Produkt der Arbeit gehört dann den Besitzerinnen des produktiven Eigentums. Sie profitieren daher von dieser Arbeit.
-	    * Der Besitz von produktivem Eigentum verpflichtet die, die es nicht haben zum Dienst für die, die denen es gehört. 
-	    * Umgekehrt: Wenn die Besitzerin von produktivem Eigentum keinen Vorteil aus der Anwendung fremder Arbeit für sich sieht, dann gibt
-    es keine Arbeitsplätze.
+	    * Das Produkt der Arbeit gehört dann den Besitzerinnen des produktiven Eigentums, da sie die Arbeitskraft eingekauft haben. Für sie ist der Einkauf der Arbeitskraft eine profitable Sache.
```

Comment:
hier klang es so als würden sich die kapitalisten das produkt aneignen, in wirklichkeit gehört es ja schon ihnen. ist die hauptänderung würd ich sagen.